namui/src/namui/render/rendering_tree/visit.rs

```diff
@@ -79,7 +79,7 @@ impl RenderingTree {
                         xy.y -= absolute.y;
                     }
                     SpecialRenderingNode::Rotate(rotate) => {
-                        xy = rotate.get_counter_wise_matrix().transform_xy(xy);
+                        xy = rotate.get_matrix().transform_xy(xy);
```

Comment:
rotate = ccw(counter-clock-wise). so it could be cw(clock-wise).

Look at the other code, they minus or divide values. It mean this code should calculate in reversed way.

Reply:
Was it ccw? I thought it was cw until now. Because in JavaScript, it's cw.

Reply:
UH!! WAIT

Reply:
You are right, why I thought it is ccw? shit, wait please